lib/widgets/booking/bookings_table.dart

```diff
@@ -1,48 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 import 'package:cabin_booking/constants.dart';
 import 'package:cabin_booking/model/cabin_manager.dart';
 import 'package:cabin_booking/model/day_handler.dart';
 import 'package:cabin_booking/widgets/booking/bookings_stack.dart';
 import 'package:cabin_booking/widgets/layout/current_time_indicator.dart';
 import 'package:cabin_booking/widgets/layout/stripped_background.dart';
-import 'package:cabin_booking/widgets/layout/time_column.dart';
-import 'package:flutter/material.dart';
-import 'package:provider/provider.dart';
 
 class BookingsTable extends StatelessWidget {
   const BookingsTable();
 
   @override
   Widget build(BuildContext context) {
-    return Stack(
-      children: [
-        const StrippedBackground(
-          startTime: kTimeTableStartTime,
-          endTime: kTimeTableEndTime,
-        ),
-        Consumer2<DayHandler, CabinManager>(
-          builder: (context, dayHandler, cabinManager, child) {
-            return Row(
-              mainAxisSize: MainAxisSize.max,
-              mainAxisAlignment: MainAxisAlignment.start,
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                child,
-                for (final cabin in cabinManager.cabins)
-                  SizedBox(
-                    width: kColumnWidth,
-                    child: BookingsStack(
-                      key: Key('${cabin.number}'),
-                      cabin: cabin.simplified(),
-                      bookings: cabin.allBookingsOn(dayHandler.dateTime),
+    return LayoutBuilder(builder: (context, constraints) {
+      return Stack(
+        children: [
+          const StrippedBackground(
```

Comment:
We need to have two `StrippedBackground` and `CurrentTimeIndicator` in both `bookings_table.dart`and `time_column.dart` because Horizontal data table widget makes us separate the code between the main column and the rest of the columns.

Reply:
It looks a little bit awkward to me just with just showing the `CurrentTimeIndicator` over the `BookingsTable` It looks like it's being cropped, specially when you scroll the events area to the right.
Moreover, showing the current time number over the `BookingsTable` indicator will cover the first column event cards.

I don't have an strong opinion about this, but maybe as it is it is the best option. WDYT?

Reply:
Yes, you’re right that it looks better now. The other thing that concerns me is performance, as the `CurrentTimeIndicator` will be rendered twice every 5 seconds—it may be insignificant.

Reply:
What do you think if we try it as it is and if we find some problems in the future we look for a workaround?

If it is ok for you, then you can continue with the PR conflicts and merge the PR once ready 😉